content/en/graphing/widgets/_index.md

```diff
@@ -33,9 +33,23 @@ aliases:
     {{< nextlink href="/graphing/widgets/top_list" >}}Top List{{< /nextlink >}}
 {{< /whatsnext >}}
 
+## Full screen
+
+Most widgets have a full screen mode. To access this view, click on the full-screen button in the upper right-hand side of the graph.
+
+In full screen mode, you can:
+
+* Change time frames
+* Move backward or forward by the time frame selected
+* Pause the graph at the current time or view the live graph
+* Reset the time frame
+* Export the graph to a dashboard, notebook, or copy the query
+
+Additional options are available for [timeseries widgets][1].
```

Comment:
I think we should introduce the other options available, like applying [functions](https://docs.datadoghq.com/graphing/functions/#apply-functions-optional)

Reply:
You can only apply functions to timeseries graphs / widgets, so I added this info to the timeseries widget page.